packages/fast-animation/package.json

```diff
@@ -13,13 +13,15 @@
   },
   "main": "dist/index.js",
   "scripts": {
+    "build": "tsc -p ./lib/tsconfig.json",
     "build:app": "webpack --config webpack/webpack.react.config.js",
-    "prepare": "tsc -p ./lib/tsconfig.json",
+    "prepare": "npm run build",
     "start": "node_modules/.bin/webpack-dev-server --config webpack/webpack.react.config.js --progress",
     "test": "npm run unit-tests && npm run build:app",
     "tslint": "tslint -c ./tslint.json '**/*.ts{,x}'",
     "tslint:fix": "tslint -c ./tslint.json --fix '**/*.ts{,x}'",
-    "unit-tests": "jest"
+    "unit-tests": "jest",
+    "watch": "npm run build -- -w"
```

Comment:
any reason we dont need preserveWatchOutput on this?

Reply:
nope, I'll add it